packages/eslint-configs/package.json

```diff
@@ -1,6 +1,7 @@
 {
-  "name": "eslint-config-local",
+  "name": "@sentry-internal/eslint-config",
```

Comment:
Does this still work? IIRC you said yesterday that the package needs to be named `eslint-config` (?)
Or does it work b/c scope doesn't count?